osbuild/qemu.py

```diff
@@ -0,0 +1,328 @@
+import io
+import json
+import os
+import platform
+import signal
+import socket
+import subprocess
+import tempfile
+import time
+from typing import Dict, List, Optional, Tuple
+
+
+class Virtiofsd:
+    def __init__(
+        self,
+        share_dir: str,
+        socket_path: str,
+        readonly: bool,
+    ):
+        self.share_dir = share_dir
+        self.cmd = [
+            "/usr/libexec/virtiofsd",
```

Comment:
On some distros (distros which don't use `/usr/libexec`), this is installed at `/usr/lib/`.  Let's add `/usr/lib` as a fallback path if it's not in libexec.